.github/workflows/copilot-setup-steps.yml

```diff
@@ -19,3 +19,5 @@ jobs:
 
       - name: Restore solution
         run: ./build.sh --restore --excludecibinarylog --warnaserror false /p:BuildAllConfigurations=true /p:DotNetBuildAllRuntimePacks=true
+      - name: Put dotnet on the path
+        run: echo "PATH=$PWD/.dotnet:$PATH" >> $GITHUB_ENV
```

Comment:
This will be the version that's brought down as part of the above build.sh, right?

Reply:
Yep